test/transcoding/kernel_arg_type_qual.ll

```diff
@@ -0,0 +1,37 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -o - -spirv-text | FileCheck %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+; RUN: llvm-dis < %t.rev.bc | FileCheck %s --check-prefix=CHECK-LLVM
+
+; ModuleID = 'test.cl'
+source_filename = "test.cl"
+target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"
+target triple = "spir64-unknown-unknown."
+
+; CHECK-SPIRV: String 12 "kernel_arg_type_qual.test.volatile,const,,"
```

Comment:
What if an argument has multiple qualifiers at the same time?

Reply:
It's possible to have restrict volatile pointer to mark that it doesn't alias with non-volatile pointer.